### DISCUSSION TITLE
When will the desktop application be updated to allow using multiple accounts?

### DISCUSSION THREAD

**Discussion by Omnia-et-nihil** [community/community#148253] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Title. People have been asking for this since 2018, why the fuck has there been absolutely no movement on that front?

**Comment by ibra-kdbra** [comment]
Because this Microsoft, no changes till you leave the platform, they made this platform a trash one as their OS, _smh_, what a fraud.
